.github/workflows/pkgdown.yml

```diff
@@ -27,15 +29,21 @@ jobs:
       - name: Install R package dependencies
         uses: r-lib/actions/setup-r-dependencies@v2
         with:
-          extra-packages: any::pkgdown, local::.
+          extra-packages: any::pkgdown, any::remotes, local::.
 
-      - name: Configure git
+      - name: Build site
+        shell: Rscript {0}
         run: |
-          git config user.name "$GITHUB_ACTOR"
-          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+          source("pkgdown/build.R")
+          build_versioned(
+            repo = ".",
+            versions = yaml::read_yaml("pkgdown/versions.yaml"),
```

Comment:
Let's use `yml` extension instead of `yaml` for consistency with all the other YAML files in the project.